core/src/backend/render.rs

```diff
@@ -146,6 +149,13 @@ impl RenderBackend for NullRenderer {
     fn activate_mask(&mut self) {}
     fn deactivate_mask(&mut self) {}
     fn pop_mask(&mut self) {}
+
+    fn get_bitmap_pixels(&mut self, _bitmap: BitmapHandle) -> Option<Bitmap> {
+        None
+    }
+    fn register_bitmap_raw(&mut self, _width: u32, _height: u32, _rgba: Vec<u8>) -> BitmapHandle {
```

Comment:
This should probably return a `Result` if it could fail on some backends. This can avoid the `Results` later on.